package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "eslint-plugin-folders",
-  "version": "1.0.3",
+  "name": "eslint-plugin-folders2",
+  "version": "0.0.1",
```

Comment:
lets not rename the plugin or reset the version

Reply:
I changed this